frontend/src/redux.js

```diff
@@ -26,11 +37,14 @@ const configureStore = initialState => {
   }
 
   const store = createStore(
-    reducer,
+    // reducer,
+    persistedReducer,
     initialState,
     composeEnhancers(applyMiddleware(...middlewares))
   );
   return store;
 };
 
-export default configureStore;
+let store = configureStore();
+let persistor = persistStore(store);
```

Comment:
These two should be consts

Reply:
Not necessarily, even the official docs use let. :)

Reply:
Yeah that's fair enough. However, the linting comes up with the error:
<img width="678" alt="screen shot 2018-10-02 at 09 42 25" src="https://user-images.githubusercontent.com/11047071/46338254-8ad46680-c627-11e8-807d-3b4b28869f7b.png">